### PR TITLE
feat(STONEINTG-811): Add CA certs for internal deployments

### PR DESCRIPTION
**Pull request by MartinBasti**
Integration-service needs CA cert to work with internal deployments

### CODE REVIEW

**Comment by MartinBasti** [comment]
Moving to draft, I had discussion with Gal, it needs rework to use different method. This way is deprecated.

**Comment by MartinBasti** [comment]
I have to debug why it's out of sync, didn't experience this on my local cluster
```
integration-in-cluster-local                      OutOfSync   Healthy
```

**Comment by MartinBasti** [comment]
Fixed issue with argoCD

**Review by dirgim — APPROVED** [review]
/lgtm
/approve

**Review by dirgim — APPROVED** [review]
/lgtm
/approve

File: components/integration/production/stone-prod-p01/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../base
+components:
+  - ../../rh-certs
```
Comment (hongweiliu17) [inline components/integration/production/stone-prod-p01/kustomization.yaml:6]: I can't find the files under `../../rh-certs`, am I wrong? Thanks!

Comment (MartinBasti) [reply components/integration/production/stone-prod-p01/kustomization.yaml:6]: image
I see them there

Comment (hongweiliu17) [reply components/integration/production/stone-prod-p01/kustomization.yaml:6]: Ah, yes, thanks. I found it.

**Review by hongweiliu17 — APPROVED** [review]
LGTM

**Pull request merged**